internal/app/components/plans/plans.go

```diff
@@ -6,7 +6,9 @@ import (
 
 type Plan interface {
 	IsCompleted(records *courses.CourseRecords) bool
+	GetName() string
 }
+type Plans []*Plan
```

Comment:
you don't have to listen to me since i don't know much about this, but i'm not sure if we should be using interface pointers (https://medium.com/@saiyerram/go-interfaces-pointers-4d1d98d5c9c6)